utils/docker-ce/files/dockerd.init

```diff
@@ -3,12 +3,41 @@
 USE_PROCD=1
 START=25
 
+DOCKERD_CONF="/tmp/dockerd/daemon.json"
+
+handle_registry_mirror() {
```

Comment:
This looks like it would be useful in a general sense when more settings are added as well. 
we could probably rename it to something like `json_add_array_string`